### ISSUE TITLE
Using a fixed seed doesn't seed to work

### ISSUE THREAD

**Issue by mjeensung** [wzhouad/ATLOP#5]
The random seed is fixed to 66 in the code, but the result slightly changes even in the same setting.

Is it possible to see the same results every run?

**Comment by wzhouad** [comment]
I don't know how to make the code fully deterministic. When I run the code with the same seed, the loss differs after a few steps and accumulates in training, leading to different results. I think it's a common problem in Pytorch.

**Comment by mjeensung** [comment]
It's interesting.
Thank you for your answer.

**Comment by Lanyu123** [comment]
Thank you sir, one more question，Since the best results are different with same random seed, the results mentioned in your paper  are trained with the same random seed (like 66)  or  trained with different random seed and averaged them?
